src/ui/components/shared/UserSettingsModal/ExperimentalSettings.tsx

```diff
@@ -64,6 +64,11 @@ const EXPERIMENTAL_SETTINGS: ExperimentalSetting[] = [
     description: "Cache the results of indexing the recording",
     key: "cacheScanData",
   },
+  {
+    label: "New Controller on Refresh",
```

Comment:
I don't like leaking the concept of a controller into the frontend, but since this is buried in the experimental settings, I feel ok about it.